sup3r/pipeline/forward_pass.py

```diff
@@ -794,6 +722,34 @@ def __init__(self, file_paths, model_kwargs, fwp_chunk_shape,
 
         self.preflight()
 
+    def get_input_handler_kwargs(self, input_handler_kwargs):
+        """Get input handler args from input_handler_kwargs dict
+
+        Parameters
+        ----------
+        input_handler_kwargs : dict
+            Dictionary of args to pass to the data handler
+            :class:`sup3r.preprocessing.data_handling.DataHandler`
+        """
+        self._target = input_handler_kwargs.get('target', None)
+        self._grid_shape = input_handler_kwargs.get('shape', None)
+        self.raster_file = input_handler_kwargs.get('raster_file', None)
+        self.temporal_slice = input_handler_kwargs.get('temporal_slice',
+                                                       slice(None))
+        self.time_chunk_size = input_handler_kwargs.get('time_chunk_size',
+                                                        None)
+        self.overwrite_cache = input_handler_kwargs.get('overwrite_cache',
+                                                        False)
+        self.overwrite_ti_cache = input_handler_kwargs.get(
+            'overwrite_ti_cache', False)
+        self.extract_workers = input_handler_kwargs.get('extract_workers',
+                                                        None)
+        self.compute_workers = input_handler_kwargs.get('compute_workers',
+                                                        None)
+        self.load_workers = input_handler_kwargs.get('load_workers', None)
+        self.ti_workers = input_handler_kwargs.get('ti_workers', None)
+        self._cache_pattern = input_handler_kwargs.get('cache_pattern', None)
```

Comment:
There's an unofficial pep8 standard that says all attributes should be instantiated in the `__init__`. I've found that doing this makes it a lot easier to track attributes and figure out what they are / where they're defined or modified. Thoughts on moving all of these definitions back up, or making them properties that just retrieve the desired attr from `self.input_handler_kwargs` dynamically?

Reply:
Yeah i got the feeling this was not standard. Will move back up to  init.

Reply:
Yeah the main thing that sucks if the attrs arent initialized in `__init__` is when you see an attr somewhere random and you cant figure out where it was actually set.

Reply:
Yeah definitely makes sense. Were doing this in ForwardPass.get_chunk_kwargs also though. I wanted to move a lot of that to properties so I suppose now is the time.

Reply:
Yeah totally your call if you move them to properties or just attributes set in `__init__`. If you move them to properties just make sure to say "retrieved from `input_handler_kwargs` with default of ..." in the docstring